Sources/GRPC/PlatformSupport.swift

```diff
@@ -127,7 +128,11 @@ extension ClientBootstrap: ClientBootstrapProtocol {}
 
 #if canImport(Network)
 @available(OSX 10.14, iOS 12.0, tvOS 12.0, watchOS 6.0, *)
-extension NIOTSConnectionBootstrap: ClientBootstrapProtocol {}
+extension NIOTSConnectionBootstrap: ClientBootstrapProtocol {
```

Comment:
We need to provide a default implementation on `ClientBootstrapProtocol` and `ServerBootstrapProtocol` as well (otherwise this would be a breaking change). 

We could also just remove the implementation on the `NIOTS*Bootstrap`s and defer to the default ones.